sql/payday.sql

```diff
@@ -55,7 +55,7 @@ CREATE TABLE payday_payments_done AS
 
 DROP TABLE IF EXISTS payday_payment_instructions;
 CREATE TABLE payday_payment_instructions AS
-    SELECT participant, team, amount
+    SELECT s.id, participant, team, amount, due
       FROM ( SELECT DISTINCT ON (participant, team) *
```

Comment:
I've made all changes except for this particular one, because I'm not sure about the filtering by ```ts_start```. As I see it, ```payment_instructions``` can only have a record with ```mtime``` > ```ts_start``` if a new subscription is created after the payday run has been started. And if this situation comes up, we do have a problem - the ```current_payment_instructions``` view will pick up this latest record, while ```payday_payment_instructions``` (as it is now) will not. If we do update the subquery, this could have different results depending on the exact point in payday when the new subscription is created.

Once again, I've got a 'diverged' situation after rebasing with master:
>Your branch and 'origin/charge-in-arrears' have diverged,
and have 196 and 14 different commits each, respectively.

However, this seems ok after checking the log - will just have to do a forced push like last time.